docs/user/metal/install_ipi.md

```diff
@@ -195,3 +195,80 @@ network configuration, but fully supporting alternative subnets for the
 provisioning network is incomplete.
 
 https://github.com/openshift/installer/issues/2091
+
+## Troubleshooting
+
+General troubleshooting for OpenShift installations can be found
+[here](/docs/user/troubleshooting.md).
+
+### Bootstrap
+
+The bootstrap host is currently a libvirt VM running on the same host as
+the installer. This bootstrap host runs the Ironic services needed to
+provision the control plane. Ironic being available is dependent on
+having successfully downloaded the RHCOS and Ironic agent images. In
+some cases, this may fail, and the installer will report a timeout
+waiting for the Ironic API.
+
+You can view the Ironic logs by sshing to the bootstrap node, and
```

Comment:
We could mention how to SSH to the VM, e.g via inspection of virsh leases, or using the provisioning IP, or the API VIP - probably using the leases is the safest since it's always available, even if things failed very early and keepalived/ironic things didn't start?